docs/contribution.md

```diff
@@ -0,0 +1,157 @@
+# Contribution Guidelines
+
+## Ground rules & expectations
+
+Before we get started, here are a few things we expect from you (and that you should expect from others):
+
+*   Be kind and thoughtful in your conversations around this project. We all come from different backgrounds and
+    projects, which means we likely have different perspectives on "how open source is done." Try to listen to others
+    rather than convince them that your way is correct.
+*   Please ensure that your contribution passes all tests. If there are test failures, you will need to address them
+    before we can merge your contribution.
+*   When adding content, please consider if it is widely valuable. Please don't add references or links to things you or
+    your employer have created as others will do so if they appreciate it.
+*   When reporting a vulnerability on the software, please, put in contact with IoT Agent Node Lib repository maintainers in order to discuss it 
+    in a private way.
+
+## How to contribute
+
+If you'd like to contribute, start by searching through the [issues](https://github.com/telefonicaid/sigfox-iotagent/issues) and
+[pull requests](https://github.com/telefonicaid/sigfox-iotagent/pulls) to see whether someone else has raised a similar idea or
+question. In adition, you can also check in the IoTAgent Node Lib framework repository for [issues](https://github.com/telefonicaid/iotagent-node-lib/issues) and [pull requests](https://github.com/telefonicaid/iotagent-node-lib/pulls) across all the IoT-Agents
+
+If you don't see your idea listed, and you think it fits into the goals of this guide, do one of the following:
+
+-   **If your contribution is minor,** such as a typo fix, open a pull request.
+-   **If your contribution is major,** such as a new guide, start by opening an issue first. That way, other people can
+    weigh in on the discussion before you do any work.
+
+### Pull Request protocol
+
+As explained in ([FIWARE Development Guidelines](https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Developer_Guidelines)) 
```

Comment:
Broken link?

(Not sure if in other similar PRs we took the decision of using this link even broken and to be solved in the future)

Reply:
Similar in other PRs, we decided to use this link while finding the fixed link / another alternative